src/components/Proposal/View/ProposalGovernanceSection.tsx

```diff
@@ -81,14 +93,14 @@ function ProposalResultSection({
         {showResults && (
           <div
             className={TokenList.join([
-              'ProposalResultSection__Results',
-              !finished && 'ProposalResultSection__Results--current',
+              'ProposalGovernanceSection__Results',
+              !finished && 'ProposalGovernanceSection__Results--current',
             ])}
           >
-            {results.map((result) => {
+            {partialResults.map((result) => {
               return (
                 <ChoiceProgress
-                  onClick={onOpenVotesList}
+                  // onClick={onOpenVotesList} TODO
```

Comment:
Should we remove this?

Reply:
I missed this, tx. Changed it for `onClick={() => updatePageState({ showVotesList: true })}`